### ISSUE TITLE
Hung printing dots after "Starting automated install....."

### ISSUE THREAD

**Issue by bwann** [rhinstaller/anaconda#953]
During a kickstart install of CentOS, in some cases Anaconda hangs and doesn't proceed. On the tmux console session I'll see "`Starting automated install..........`" and it'll just sit there printing dots to console forever.  I haven't been able to repro it reliably, I just catch it by chance; I can reinstall the same hardware over and over and it'll work as expected otherwise.

Because things end abruptly in storage.log, this makes me think it's a blivet problem of some kind. In the logs I'm attaching it's discovering the devicetree, but in several other cases the last entry in storage.log is "`Running... dumpe2fs -h /dev/mapper/live-rw`" with no further output. Either case there's no `dumpe2fs`, `mkfs*`, udevadm` processes actually running when it's printing endless dots.

I can't force a traceback file out of Anaconda, it just writes a zero-byte /tmp/anaconda-tb file. Strace of the anaconda process shows `futex(..., FUTEX_WAIT_PRIVATE, 2, NULL)` like it's waiting on a thread.  Logs show pre-install scripts all run successfully.

Any ideas? I'm happy to build with more debugging to see if we can get better data.

Anaconda 26.1-1, built from github at commit bab1f5f5b on 2016-12-15
Blivet 2.1.7
Fedora Rawhide stage2 built with Lorax on 12/15, installing CentOS 7

Various {anaconda, program, storage}.log files and other output from affected system:
https://gist.github.com/bwann/1511f7993185a89b992c0decb8bdcb7c

**Comment by dychan** [comment]
Just for reminding further reader. In my case, the file name of repository data file under repodata/ directory was trancated. This result in not usable installation media repository. To fix it, I build custom ISO file with directive "-J -joliet-long" aganst mkisofs command.

**Comment by nipakoo** [comment]
@bwann Any updates? Did you figure out what was causing this? We are facing similar occasional hangs in our Koji image builds.

**Comment by bwann** [comment]
@nipakoo Nope, I never figured out what was causing it. The short term hack was to have part of our provisioning system scrape the console output and reboot/restart things when it happened.

**Comment by nipakoo** [comment]
Ok thanks anyway :) guess we'll go for something similar for now

**Comment by sregister** [comment]
Has anyone found a solution to this other than detecting the dots and reinstalling?

**Comment by jkonecny12** [comment]
Could you please move this bug and the discussion to the [bugzilla](bugzilla.redhat.com)?

**Comment by poncovka** [comment]
Thank you for reporting this issue! The issue tracker will be closed on 3 June 2019. Please make sure that your issue is also reported at https://bugzilla.redhat.com/. We are handling a high number of issues and many of them involve other components. Bugzilla allows us to better process your issues and collaborate on them with other teams. We are sorry for the inconvenience.
